backends/conrod_wgpu/src/lib.rs

```diff
@@ -537,8 +539,9 @@ fn default_image_tex_desc() -> wgpu::TextureDescriptor<'static> {
     }
 }
 
-fn sampler_desc() -> wgpu::SamplerDescriptor {
+fn sampler_desc<'a>() -> wgpu::SamplerDescriptor<'a> {
```

Comment:
Just a nitpick, but I think this function could just return `wgpu::SamplerDescriptor<'static>` so you don't need the lifetime parameter `'a`.

Reply:
Thanks so much, probably I don't understand how to mark lifetimes properly... (I'm still a newbie to Rust)